source/styles/layout/_search.scss

```diff
@@ -0,0 +1,65 @@
+.slider {
+    margin: 0.5rem 0;
+}
```

Comment:
mieszasz slider i search w jednym pliku , slider leci do pliku slider.scss ale tutaj raczej powinno być 
`.search {
    &__header { ... }
    &__content { ... }
    &__body { ... }
}`

Nie ma zachowanego BEMa oraz OOCSS  <- do przestudiowania to :)